uiowa/uids
# Tabs component styling changes

Tracks #629 

# How to test

- Test with `bg--black`, `bg--gold`, and `bg--gray` background classes on the `<body>` and verify that tab hover/active states remain with accessible color contrast. 
- Test that you can navigate with keyboard only and verify that focus styles provide sufficient indication of where you are within the tabs.
